packages/conf-libogg/conf-libogg.1/opam

```diff
@@ -3,7 +3,7 @@ maintainer: "https://github.com/ocaml/opam-repository/issues"
 homepage: "https://xiph.org/ogg/"
 bug-reports: "https://github.com/ocaml/opam-repository/issues"
 authors: "Xiph.Org Foundation"
-license: "BSD"
+license: "BSD-1-Clause"
```

Comment:
It appears BSD-3 here: https://gitlab.xiph.org/xiph/ogg
butI don't know which variant I should pick